pingcap/pd
# Add TiDB TLS config for tidb-dashboard

### What problem does this PR solve? 

Add TLS config for tidb-dashboard to connect to TiDB

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)
1. Enable mTLS authentication by specifying `ssl-ca`, `ssl-key`, `ssl-cert` on TiDB
2. Add `tidb-cacert-path`, `tidb-cert-path` and `tidb-key-path` on pd config file.
```
[security]
  tidb-cacert-path = "/path/ca.pem"
  tidb-cert-path = "/path/client.pem"
  tidb-key-path = "/path/client-key.pem"
```
3. Check the if the embedded tidb-dashboard works 

Code changes

 - Has configuration change
Add three more config field on `security` section:`tidb-cacert-path`, `tidb-cert-path` and `tidb-key-path`


Related changes

https://github.com/pingcap-incubator/tidb-dashboard/pull/219
